### DISCUSSION TITLE
Raspberry pi4 SSD full - Blockchain too big?

### DISCUSSION THREAD

**Discussion by keyglow** [Chia-Network/chia-blockchain#11126] [Support]
Hello ,
Since a few days the Chia Client has stopped farming. Because the hard disk is full again. 
My "blockchain_v1_mainnet.sqlite" file is 93gb big.

**Have I maybe configured something wrong ? Or i need to upgrade on a 256gb sd?**
![big-blockchain](https://user-images.githubusercontent.com/84075465/162710798-e260a8f2-2055-4d98-8e57-fbd277014b4b.PNG)

**Comment by loppefaaret** [comment]
However you look at it, at some point, you are likely going to need a bigger drive to host the database file
for now, you could try and do a db upgrade, read more here:
https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference#db
